### DISCUSSION TITLE
DMS primary key

### DISCUSSION THREAD

**Discussion by sojan1** [Mimetis/Dotmim.Sync#1088] [Q&A] [answered]
Hi,

Does DMS identifies  primary key and synchronize records based on the primary keys in each SQL Server table ?
![image](https://github.com/Mimetis/Dotmim.Sync/assets/111710684/1f787f6d-3939-4a59-baa1-d76effbd2883)

**Comment by Mimetis** [comment, marked as answer]
yes
